### ISSUE TITLE
Dokumentation/Hardware?

### ISSUE THREAD

**Issue by username7291** [lucadentella/ArduinoLib_CEClient#4]
How to use this? The only included example seems to monitor the communication between two other existance devices.
It points to another Project: https://github.com/floe/CEC, that point again to another thats 404.
Is this strange complex Hardware of floe/CEC used?
There exists only one CEC line but 2 I/O are wasted. At 3V3 Adrduino no Hardware should required.
On a 5V Arduino a single FET is would required for Voltage conversion.
I assume the complex Hardware convert the HIGH to LOW and LOW to HIGH?
Dont get it the code to work with 1 Pin. All 5 trys the TV find 1 to 6 different devices that are not set, never incoming messages.

**Comment by lucadentella** [comment]
Hi! You can use the interface designed by floe: I was able to monitor CEC communcation and also to turn a device ON/OFF from Arduino (sending the "standby" command). 

A good source for CEC information is CEC-o-matic: https://www.cec-o-matic.com/

I see your point about "wasting" two pins while one should be sufficient... at the moment the library uses two pins and the external interface to convert the voltage and the logic levels (high->low...). If you have time to optimize it, I'd be glad to evaluate your code, just open a PR

thanks

**Comment by username7291** [comment]
I modified to work it with 1 single pin, no additional Hardware required(Arduini Pro Mini 3V3 8MHZ) exept a additional Pullup of 3K3, but i think its not required, internal Pullup should work. Problem was the missing documentation, that cost me 3 days to find out that the modified Version works. The ceclient.run(); must called all the time, you cant do much between, or there arrive no data&no data gets send.  I got then some Data In, but thinked this Software cant send, cause the software this is based on has issues that tells it cant. Would better block writes until ready and read on a interrupt.
Then i found out that the Data is not send like in https://www.cec-o-matic.com. Its without the first byte which belongs to address instead. Then this stupid Samsung Television not Display OSD Messages, why ever.
